### PR TITLE
Added aria-label to buttons

### PR DESCRIPTION
**Pull request by harrington101**
Added aria-label to buttons so that the viewer passes WAVE Accessibility.

### CODE REVIEW

**Comment by harrington101** [comment]
Added aria-label to buttons so that the viewer passes WAVE Accessibility.

File: src/ImageGallery.jsx [hunk 1]
```diff
@@ -125,6 +128,7 @@ export default class ImageGallery extends React.Component {
       return (
         <button
           type='button'
+          aria-label='Toggle Full Screen Mode'
```
Comment (xiaolin) [inline src/ImageGallery.jsx]: This could be `{isFullscreen ? 'Exit Full Screen' : 'Enter Full Screen'}`.

Comment (harrington101) [inline src/ImageGallery.jsx]: Good idea. I've added it.

**Comment by xiaolin** [comment]
@harrington101 I notice something felt off, the aria labels has already been added from a previous commit. What version of image gallery are you on?

**Comment by xiaolin** [comment]
You pr is creating duplicate aria labels
```
    renderLeftNav: (onClick, disabled) => {
      return (
        <button
          type='button'
          aria-label='Previous Slide'
          className='image-gallery-left-nav'
          disabled={disabled}
          onClick={onClick}
          aria-label='Previous Slide'
        />
      );
    },
    renderRightNav: (onClick, disabled) => {
      return (
        <button
          type='button'
          aria-label='Next Slide'
          className='image-gallery-right-nav'
          disabled={disabled}
          onClick={onClick}
          aria-label='Next Slide'
        />
      );
```

**Comment by harrington101** [comment]
I'm on 0.7.12

**Comment by xiaolin** [comment]
0.7.13 has the aria-labels that @Josephjbliss added in #141

**Comment by harrington101** [comment]
OK. I guess I'm a version behind. Thanks though. Thanks @Josephjbliss too.

**Pull request closed without merging**